lightwood/encoders/time_series/helpers/rnn_helpers.py

```diff
@@ -7,17 +6,14 @@
 
 def tensor_from_series(series, device, n_dims, pad_value, max_len=None, normalizer=None):
     """
-    :param series: list of lists, corresponds to time series: [[x1_1, ..., x1_n], [x2_1, ..., x2_n], ...]
+    :param series: list of lists, corresponds to a time series: [[x1_1, ..., x1_n], [x2_1, ..., x2_n], ...]
                    the series is zero-padded on each axis so that all dimensions have equal length
```

Comment:
Is the series zero-padded as part of this method, or should the series being passed already be zero-padded ?

It seems to be padded in here, so maybe mention that (or rather, make it clearer that it's the case as part of the comment)

Reply:
Yup, padded here, will update docstring.